Local/Factoids.py

```diff
@@ -1,14 +1,19 @@
+"""Simple file to deal with static factoids"""
```

Comment:
Use docstrings inside functions to describe their purpose, inputs, and outputs. Else use `#`